app/assets/stylesheets/pages/home/_subscribe.scss

```diff
@@ -38,5 +50,17 @@
 	    	background-color: #fff;
 	    	color: #484848;
 	    }
+
+	    @media only screen and (max-width: 900px){
```

Comment:
try to change the font when it's hit 768px and 480px

Reply:
okay, already to change this